components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -7,6 +7,7 @@ module brave_wallet.mojom;
 
 import "mojo/public/mojom/base/time.mojom";
 import "mojo/public/mojom/base/values.mojom";
+import "url/mojom/origin.mojom";
```

Comment:
FYI iOS's framework has no equivalent of `url::Origin` and so we currently do not support Obj-C code-gen for it.  I will have to look into this but at the moment this PR will not compile for iOS